packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -667,6 +676,22 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         return _getFeeAdjustedCollateral(rawTotalPositionCollateral);
     }
 
+    /**
+     * @notice Accessor method to calculate a transformed price using the provided finanicalProductLibrary specified
+     * during contract deployment. If no library was provided then no modification to the price is done.
+     * @param price input price to be transformed.
+     * @return transformedPrice price with the transformation function applied to it.
+     */
+
+    function transformPrice(FixedPoint.Unsigned memory price) public view returns (FixedPoint.Unsigned memory) {
```

Comment:
Why can't this be an internal method and removed from the interface?

Reply:
I left this as a public method as it's pretty useful to be able to transform the price as a caller. For example, the front end of `emp-tools` would need to transform the price based on this method to "see" what the current price is, based off `emp-tools` belief of the current price.

We could leave this as internal but then calling this method becomes a bit of a pain. you'd need to then need to instantiate an instance of the library and then call this method from the context of the EMP to get the transformed price.

Reply:
Ah got it. Maybe add a comment that a monitoring party tracking the oracle price would also need access to this `transformPrice` method to derive the "price" that the FP sees.